### PR TITLE
Holiday Component

### PR DESCRIPTION
**Pull request by pkadatare**

### CODE REVIEW

File: src/services/leave-service.ts [hunk 1]
```diff
@@ -20,4 +22,10 @@ export class LeaveService {
     let url = this.leaveServiceUrl + "/available?employeeId=" + id;
     return this.http.get(url).map(res => res.json());
   }
+  
+  getAllHolidays():Observable<PublicHoliday[]>{
+    let url = this.leaveServiceUrlHoliday ;
+    console.log(url);
```
Comment (sskelkar) [inline src/services/leave-service.ts]: Please remove this console.log

File: src/components/public-holidays/public-holidays.tpl.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+You have following Publie Holidays :<br/>
```
Comment (sskelkar) [inline src/components/public-holidays/public-holidays.tpl.html]: "You have following Publie Holidays :" why not write something like "Public holidays in current year: "

Comment (mbgangurde) [inline src/components/public-holidays/public-holidays.tpl.html]: yes, boss is right.

Comment (sskelkar) [inline src/components/public-holidays/public-holidays.tpl.html]: (envy)

**Pull request merged**